infra/core/managed-cluster/.bicep/app-config-data-reader.bicep

```diff
@@ -0,0 +1,18 @@
+@description('Required. The parameter object for the managed identity. The object must contain the name and principalId values.')
```

Comment:
Can you please update the description, it is just string not an object.

Reply:
@ppotturi Good spot!  Copied from another bicep and didn't notice the description.  I've updated it now.